sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ViewTest.java

```diff
@@ -478,7 +478,7 @@ public void testMultimapSideInput() {
 
     final PCollectionView<Map<String, Iterable<Integer>>> view =
         pipeline
-            .apply("CreateSideInput", Create.of(KV.of("a", 1), KV.of("a", 2), KV.of("b", 3)))
+            .apply("CreateSideInput", Create.of(of("a", 1), of("a", 2), of("b", 3)))
```

Comment:
In this case it isn't readable as `of`, so wherever `KV.of` is imported statically should be switched to non-static import and written as `KV.of`.